test/string-checker.js

```diff
@@ -356,4 +356,24 @@ describe('modules/string-checker', function() {
             });
         }
     });
+
+    describe('getRegisteredPresets', function() {
+        it('returns the configuration\'s registered presets', function() {
```

Comment:
There's already a test for getRegisteredPreset in configuration and this function's sole role is to delegate to that tested method. Doing checks on the returned value is redundant.

What's the minimum checking here that you feel is appropriate?